Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js

```diff
@@ -0,0 +1,59 @@
+// call the displayWindowProperties on page loads
+document.addEventListener("DOMContentLoaded", function () {
+  displayWindowProperties();
+});
+
+function displayWindowProperties() {
+  // get the userAgent property from the window properties
+  const userAgent = window.navigator.userAgent;
+  console.log("displayWindowProperties ~ userAgent:", userAgent);
+
+  // get the screenWidth property from the window properties
+  const screenWidth = window.screen.width;
```

Comment:
an single const can be created for height and width both

Reply:
const screenWidthAndHeight = `${window.screen.width} x ${window.screen.height}`;